client/web/admin/src/components/Authclient/CAuthclientEditorInfo.vue

```diff
@@ -316,11 +325,12 @@
                   ref="cUrl"
                   data-test-id="cURL"
                 >
-curl -X POST {{ curlURL }} \
--d grant_type=client_credentials \
--d scope='profile api' \
--u {{ resource.resourceID }}:{{ secret || 'PLACE-YOUR-CLIENT-SECRET-HERE' }}
+                  curl -X POST {{ curlURL }} \
```

Comment:
This has to be misaligned since its a pre, check before and after this change and you'll see what I mean